src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/Issues-and-Incident-management-and-response.mdx

```diff
@@ -44,35 +46,15 @@ feed."
 
 ## Search and filter issues [#search-filters]
 
-You can easily search for any issue by using the search bar free text. This search bar enables you to search by:
-
-- Issue name
-- Issue ID
-- Policy Name
-- Condition Name
-- Entity name
-- Entity ID
-
-If you click the filter icon, you can filter by any issue attributes or any related tag:
-
-- Issue state (created, active, or closed)
-- Issue acknowledged(true or false)
-- Issue muted (true or false)
-- Issue correlated (true or false)
-- Issue priority (low, medium, high, or critical)
-- Issue source (NR alerts, NR anomalies, or REST API)
-- Entity
-- Alert policy
-- Alert condition
-- Tags (search for any tag related to the issue)
+You can search for any issue with the search bar free text. You can filter by any issue attributes or any related tag by clicking the filter icon. 
```

Comment:
Are h2s the best format here? I might consult with Austin but I think the visual problem I'm having with the doc is that the sections almost seem too short to support so many h2s as the main component. Just something to think about.